src/app/calendar-app/event-editor-dialog.component.ts

```diff
@@ -60,36 +104,253 @@ export class EventEditorDialogComponent {
         this.calendars = data['calendars'];
         this.calendarFC.setValue(this.calendars[0].id);
 
-        if (data['event']) {
-            this.event = data['event'];
+        this.event = data['event'];
+        if (!data['is_new']) {
             this.calendarFC.setValue(this.event.calendar);
             this.export_url = '/rest/v1/calendar/ics/' + this.event.id;
```

Comment:
For later: the specifics of this URL may be better off being abstracted away by RunboxWebmailAPI